moveit_setup_assistant/templates/moveit_config_pkg_template/package.xml.template

```diff
@@ -20,6 +20,7 @@
   <run_depend>joint_state_publisher</run_depend>
   <run_depend>robot_state_publisher</run_depend>
   <run_depend>xacro</run_depend>
+  <run_depend>warehouse_ros_mongo</run_depend>
```

Comment:
Yes, that is a problem. We probably shouldn't make *all* `moveit_config`s depend on warehouse_ros_mongo, out of the box.

The simplest way to remove this is to add the changes in comments, so the user can uncomment them if he needs it.
Not sure if that's what we want to do though.

Does anyone have an opinion on this?